_data/changelogs/component-header.yml

```diff
@@ -2,6 +2,13 @@ title: Header
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Allow mobile nav to stay open during page loads to prevent confusion upon closing.
```

Comment:
Updated the behavior of the mobile header navigation to stay open during page load.